src/erl.rs

```diff
@@ -652,6 +652,12 @@ fn pattern<'a>(p: &'a TypedPattern, env: &mut Env<'_>) -> Document<'a> {
             ..
         } => tag_tuple_pattern(name, args, env),
 
+        Pattern::Constructor {
+            args: _args,
+            constructor: PatternConstructor::Inline,
+            ..
+        } => todo!(),
```

Comment:
Here we can check to see if the args has 1 element and render that arg as a pattern if so, and exit using `fatal_compiler_bug` if not.

Reply:
Just using `pattern(args[0].value, env)`?

Reply:
Might need to use `swap_remove` to get it to pass the borrow checker, but something like that, yes. We would need to check the length first to ensure we get a nice error message in the event of a bug rather than an out of bounce panic.